src/main/java/bio/terra/service/snapshotbuilder/utils/AggregateBQQueryResultsUtils.java

```diff
@@ -3,9 +3,16 @@
 import bio.terra.model.SnapshotBuilderConcept;
 import bio.terra.service.snapshotbuilder.SnapshotBuilderService;
 import bio.terra.service.snapshotbuilder.query.table.Concept;
+import com.google.cloud.Tuple;
 import com.google.cloud.bigquery.FieldValueList;
 
 public class AggregateBQQueryResultsUtils {
+  public static Tuple<Integer, String> toConceptIdNamePair(FieldValueList row) {
+    return Tuple.of(
```

Comment:
Instead of using a gcloud type, I think it's better to use JDK types if possible. Using `Map.Entry` could work here
```suggestion
  public static Map.Entry<Integer, String> toConceptIdNamePair(FieldValueList row) {
    return Map.entry(
```